static/src/javascripts/projects/commercial/modules/third-party-tags.js

```diff
@@ -7,13 +7,13 @@ import { commercialFeatures } from 'common/modules/commercial/commercial-feature
 import { imrWorldwide } from 'commercial/modules/third-party-tags/imr-worldwide';
 import { imrWorldwideLegacy } from 'commercial/modules/third-party-tags/imr-worldwide-legacy';
 import { remarketing } from 'commercial/modules/third-party-tags/remarketing';
-import { ias } from 'commercial/modules/third-party-tags/ias';
+import { ias, permutive } from '@guardian/commercial-core';
```

Comment:
looks good! thinking about future iterations how this would look like, maybe we could export all the third party tags as an array from `commercial-core?`

something like
`import { third-party-tags } from '@guardian/commercial-core';`

Reply:
Would this not prevent tree-shaking?

There might be a case where some platforms need only to import a subset of the third party tags.

Reply:
💯 - i haven't really put any thought yet into how we should structure the package. my suspicion is that we'll have to change it as we go along and understand fully what it there

that said, i totally agree this current approach doesn't scale :)

Reply:
Agreed—ship, then fix!
🚢 🧰 🧯

Reply:
Is this where `"sideEffects": false` in your `package.json` of the `commercial-core` module comes in handy? Webpack should then be able treeshake unused exports (given some other criteria is met). 

https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free 
https://developers.google.com/web/fundamentals/performance/optimizing-javascript/tree-shaking